packages/loop-harness/scripts/loop_scheduler.py

```
@@ -0,0 +1,725 @@
+    down with it (#34). Callers treat "" the same as an empty result set and log the failure.
+    `--paginate` avoids silently truncating discovery at the default 30-item page (#9); with
+    `--paginate` and no `-q`/`--jq`, `gh api` writes one JSON array per page back-to-back
+    (not a single combined array), so parsing must handle multiple concatenated JSON documents
+    (see `_parse_paginated_json`).
+
+    A `timeout=30` expiry is treated the same as a nonzero exit (#F20): `subprocess.run` raises
+    `TimeoutExpired` instead of returning a `CompletedProcess`, which would otherwise violate
+    this function's "failure returns \"\"" contract and take the resident scheduler down with it.
+    """
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)